main.py

```diff
@@ -0,0 +1,102 @@
+is_exit = True
+
+
+
+class Banking:
```

Comment:
We should use proper names for classes. 

```suggestion
class User:
    mobile_number = ''
    password = ''
    balance = 0.0